libs/sql-schema-describer/src/postgres.rs

```diff
@@ -746,44 +740,22 @@ fn get_column_type(row: &ResultRow, enums: &[Enum]) -> ColumnType {
 static RE_SEQ: Lazy<Regex> = Lazy::new(|| Regex::new("^(?:.+\\.)?\"?([^.\"]+)\"?").expect("compile regex"));
 
 static AUTOINCREMENT_REGEX: Lazy<Regex> = Lazy::new(|| {
-    Regex::new(
-        r#"nextval\('(?:"(?P<schema_name>.+)"\.)?(")?(?P<table_and_column_name>.+)_seq(?:[0-9]+)?(")?'::regclass\)"#,
-    )
-    .unwrap()
+    Regex::new(r#"nextval\((\(?)'(((".+")|(.+))\.)?(("(?P<sequence>.+)")|(?P<sequence2>.+))'(::text\))?::regclass\)"#)
```

Comment:
I don't understand what this part matches: `(\(?)'(((".+")|(.+))\.)?`

Also I think `(("(?P<sequence>.+)")|(?P<sequence2>.+))'(::text\))?` doesn't need the `?` at the end (unless we expect some sequences to have no name).

golly regexes take time to read^^

Reply:
The first part would match the schema name, once with quotes and once without. But I just realized that when we're not extracting the schema name anymore `.+` would be enough without the or on `".+"`

And the ? from the second comment makes only the block with `::text)` optional. Maybe the escaped ) confused you?

Reply:
Ah yes, indeed. I don't know about the or, but it indeed feels like it is maybe redundant. Sounds good :+1: